Java/fclUserManagement/src/main/java/com/fcl/users/UserServicesImpl.java

```diff
@@ -0,0 +1,125 @@
+package com.fcl.users;
+
+import java.sql.*;
+
+public class UserServicesImpl implements UserServices {
+    private Connection connection = null;
+
+    public UserServicesImpl() {
+    }
+
+    public void connectDb() throws ClassNotFoundException {
+        Class.forName("org.sqlite.JDBC");
+
+        try {
+            // create a database connection
+            connection = DriverManager.getConnection("jdbc:sqlite:F:/Database/SQLite DB/mydb.db");
```

Comment:
need to  take care of hardcoded driver path here as well.
need to take it from an ini/xml file